goty-client/src/app/form/form.component.html

```diff
@@ -1,8 +1,11 @@
 <app-info>
```

Comment:
Should rename to `submission.component.html`.

Reply:
done